app/src/components/submissionPage/SubmissionBoxWrapper.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { SnapSubmissionColumnDiv } from "./SnapSubmissionStyles";
+
+const SubmissionBoxWrapper: React.FunctionComponent = ({ children }) => (
+    <div className="container-sm ">
```

Comment:
I realise you just copied this from elsewhere, but I'm fairly confident this should be a `col`, not another container. Bootstrap doesn't really recommend nesting containers.

```suggestion
    <div className="col">
```